scripts/docs/snippetadapter.js

```diff
@@ -27,7 +27,8 @@ module.exports = function snippetAdapter( data ) {
 		entry: data.snippetSource.js,
 		outputPath,
 		language: snippetConfig.language,
-		production: data.options.production
+		production: data.options.production,
+		templateCommentLoaderConfig: data.options.templateCommentLoaderConfig || {}
```

Comment:
Why not `smarCodeInjectus`?

Reply:
smart{CodeInject}uś

Reply:
`injectedVariables`

Reply:
But I do not think that many JS developers will find C++ names familiar.